packages/tpetra/core/test/MultiVector/MultiVector_get2dView.cpp

```diff
@@ -242,14 +242,20 @@ namespace { // (anonymous)
 
       Teuchos::ArrayRCP<Teuchos::ArrayRCP<const Scalar> > viewsConst =
         X_noncontig->get2dView ();
-      TEST_ASSERT( ! X_noncontig->need_sync_host () );
+      if (! std::is_same<typename MV::dual_view_type::t_dev::memory_space,
```

Comment:
If the host/device memory spaces are the same, shouldn't need_sync_host always return false?

Reply:
I'll take a second look at report back.